mobile/native/src/db/mod.rs

```diff
@@ -114,6 +114,23 @@ pub fn insert_order(order: trade::order::Order) -> Result<trade::order::Order> {
 
 pub fn update_order_state(order_id: Uuid, order_state: trade::order::OrderState) -> Result<()> {
     let mut db = connection()?;
+
```

Comment:
Shall we also prevent going from a final state back to an open state?

Reply:
Maybe the comment is confusing, because what you propose is what we are doing.

With the comment I just wanted to highlight that we are preventing a specific scenario that has happened to us: filling to open. But we are _also_ preventing final state to open.

Reply:
yup, got confused by the comment!

Reply:
Do adapt it if you think it's important 😃 But I need to 💤 now.